daemon_ros_client/LedColor[31] colors
